Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_aitdm.nut

```diff
@@ -1,22 +1,36 @@
 untyped
 global function GamemodeAITdm_Init
 
-const SQUADS_PER_TEAM = 3
+// these are now default settings
+const int SQUADS_PER_TEAM = 4 // was 3
 
-const REAPERS_PER_TEAM = 2
+const int REAPERS_PER_TEAM = 2
 
-const LEVEL_SPECTRES = 125
-const LEVEL_STALKERS = 380
-const LEVEL_REAPERS = 500
+const int LEVEL_SPECTRES = 125
+const int LEVEL_STALKERS = 380
+const int LEVEL_REAPERS = 500
+
+// add settings
+global function AITdm_SetSquadsPerTeam
+global function AITdm_SetReapersPerTeam
+global function AITdm_SetLevelSpectres
+global function AITdm_SetLevelStalkers
+global function AITdm_SetLevelReapers
 
 struct
 {
 	// Due to team based escalation everything is an array
-	array< int > levels = [ LEVEL_SPECTRES, LEVEL_SPECTRES ]
+	array< int > levels = [] // since we added modification should init starter level int Spawner()
```

Comment:
I'm not quite sure what that comment means... Anyone can ELI5? ^^"

Reply:
The array is initilazed later so that mods can modifiy the values using the new `AITdm_SetLevel*` functions